README.md

```diff
@@ -19,21 +19,34 @@ To install OpenCLSim, run this command in your terminal. This is the preferred m
 pip install openclsim
 ```
 
-Make sure you also install the  dependencies, which are recorded in the file `requirements.txt`.
-If you have an anaconda setup, you can use the following command:
+For local development use:
 
 ``` bash
-conda install --file requirements.txt
-```
-
-If you are using a pip based environment, you can install the requirements with pip:
-
-``` bash
-pip install -r requirements.txt
+pip install -e .
```

Comment:
For opentnsim I also added the following to keep module and testing dependencies separate:
```
pip install -e .[testing]
```

Reply:
pls add that to readme

Reply:
we should perhaps make a separate requirements.txt instead of reqs in setup.